functions/sg/deploy.sh

```diff
@@ -0,0 +1,39 @@
+# Deploy the submit_sg_workload cloud function
+# usage: bash deploy.sh TRIGGER_BUCKET_NAME
+# ex: bash deploy.sh broad-gotc-dev-wfl-sg-test-inputs
+
+TRIGGER_BUCKET_NAME=${1}
+TRIGGER_EVENT=${2:-"google.storage.object.finalize"}
+REGION=${3:-"us-central1"}
+
+# gotc-prod
+if [ "${TRIGGER_BUCKET_NAME}" == "???" ]; then
+    GCLOUD_PROJECT="broad-gotc-prod-storage"
+    SA_EMAIL="sg-submission-fn-non-prod@broad-gotc-prod-storage.iam.gserviceaccount.com"
```

Comment:
Should this be `sg-submission-fn-prod` instead of non-prod?

Reply:
It doesn't exist but none of the gotc-prod deployment config exists, so fair enough

Reply:
Oh okay! Could you make a ticket for setting up those service accounts (if there isn't one already)?